config.json

```diff
@@ -20,6 +20,17 @@
       ]
     },
     {
+      "uuid": "dc4ad67d-0fe4-0880-5a13-3aebb8759bcc3095de1",
+      "slug": "two-fer",
+      "core": false,
```

Comment:
Do you want to leave this exercise out of the new core structure?

I think with these changes it would be good to have this as the second `core` exercise, and `leap` as the third, making sure that any exercises unlocked by them don't contain anything not already explained.

If you want to merge this as is, I'd be happy to update the config.json file and any changes to exercises that might result?